### PR TITLE
Update deprecations in ASCollectionView example, enable warnings

### PR DESCRIPTION
**Pull request by austinlouden**
Related to issue #2569.

Enabled "Overriding Deprecated Objective-C Methods" in:
- ASCollectionView
- ASDKLayoutTransition
- ASDKTube
- ASDKgram
- ASMapNode

I can continue to do the rest of these if I'm on the right track.

### CODE REVIEW

File: examples/ASMapNode/Sample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -178,6 +181,21 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		0518F692ABB8E763D9C401ED /* 📦 Embed Pods Frameworks */ = {
```
Comment (maicki) [inline examples/ASMapNode/Sample.xcodeproj/project.pbxproj]: It seems like you are using an older version of Cocoapods as it has emojis in there. Now it's like `[CP]` or something like that. Can you please  revert the changes with emojis in there? Thanks!

Comment (austinlouden) [inline examples/ASMapNode/Sample.xcodeproj/project.pbxproj]: 👍

**Comment by austinlouden** [comment]
@maicki you were right, I was on 1.0.0. Just updated

**Comment by appleguy** [comment]
@austinlouden hey, thanks a bunch for doing this!
